resolwe/process/runtime.py

```diff
@@ -254,9 +254,11 @@ def get_data_id_by_slug(self, slug):
                 "More than one data object returned for slug {}".format(slug)
             )
 
-    def update_descriptor(self, **kwargs):
-        """Update sample descriptor."""
-        raise NotImplementedError
+    def update_entity_descriptor(self, annotations):
+        """Update entity descriptor from dictionary of annotations."""
+        for key, value in annotations.items():
+            single_annotation = resolwe_runtime_utils.annotate_entity(key, value)
+            print(single_annotation)
```

Comment:
You don't need to define a new variable to just use it once. It's a personal preference, but it feels cleaner to me if you put everything into a single line.

Reply:
I was just following the style used in other functions (warning, error, info...) in the same file, but I share your opinion on this matter.